wsd/DocumentBroker.hpp

```diff
@@ -638,7 +632,7 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
         std::chrono::steady_clock::time_point _lastSaveResponseTime;
 
         /// The jailed file last-modified time.
-        std::chrono::system_clock::time_point _lastFileModifiedTime;
```

Comment:
For the purpose of git-grep-ability, I wouldn't be offended if the LastFileModifiedTime didn't go as far as ModifiedTime only ;-) - let's avoid this large rename please...